commands/test/test-command.ts

```diff
@@ -156,19 +156,67 @@ export const sendQuestion = async (interaction: any) => {
 
 
         const bestMatch = await findMatchingUser(interaction.user.id, userResponses);
+        console.log(bestMatch);
+        console.log(bestMatch?.userId);
 
         if (bestMatch) {
-            interaction.user.send(`Dein bester Gesprächspartner ist: **@${bestMatch.username}**.`);
-            interaction.user.send("Als nächstes schreibst du deinem Partner, indem du auf seinen Namen auf dem Contraversum-Server klickst 👆 und ihm eine Nachricht sendest.");
-            interaction.user.send("Dies sind drei Fragen bei denen ihr euch unterscheidet:");
+            const guildId = process.env.GUILD_ID;
+            if (!guildId) {
+                console.error('GUILD_ID is not defined in .env');
```

Comment:
instead of: ```
if (!guildId) {
                console.error('GUILD_ID is not defined in .env');
``` use:` if (!guild) throw new Error('Guild not found');` as this is cleaner